app/controllers/bookings_controller.rb

```diff
@@ -51,8 +54,7 @@ def booking_params
     params.require(:booking).permit(:start_date, :end_date)
   end
 
-  # def set_item
-  #   @item = Item.find(params[:item_id])
-  #   authorize @item
-  # end
+ def set_booking
```

Comment:
the 'authorize @booking' can actually be here instead of having it written in show, edit and destroy

Reply:
the rest is ok